lib/services/saml_test.go

```diff
@@ -38,9 +39,10 @@ func TestParseFromMetadata(t *testing.T) {
 	err := decoder.Decode(&raw)
 	require.NoError(t, err)
 
-	oc, err := UnmarshalSAMLConnector(raw.Raw)
+	ctx := context.Background()
+	oc, err := UnmarshalSAMLConnector(ctx, raw.Raw)
 	require.NoError(t, err)
-	err = ValidateSAMLConnector(oc, nil)
+	err = ValidateSAMLConnector(ctx, oc, nil)
```

Comment:
Doesn't Unmarshal call Validate? Could we drop it from the test?

Reply:
Yep, I've dropped that